### DISCUSSION TITLE
raspberry pi as agent

### DISCUSSION THREAD

**Discussion by thiamata** [Security-Onion-Solutions/securityonion#13591] [Unsupported Versions]
Hi

I am looking for a kind of easy way getting the elastic agent from SO installed on a raspberry PI.

We administrate round about 100 schools and alot have some raspis running for nearly everything. To see whatś happeing on those PIs I thouht to start over with an SO Solution.

With some Windows and i386 Linux boxes and mirror ports our first testings, works fine, but not with these PIs.

Maybe this could be an additional option in the download area at the end or even only a guide line how to set it up manually, that it works like the downloadable Agent for i386/amd64 boxes.

From elastic is an agent available for arm64, but what is needed to configure, ... .. .

many thanx for help.

Thiamata

**Comment by reyesj2** [comment]
You maybe able to follow the instructions here https://www.elastic.co/guide/en/fleet/8.10/install-fleet-managed-elastic-agent.html using the arm64 agent you need. However, you will have to get the SO CA ```/etc/pki/ca.crt``` and have it trusted on each one of those PIs. 

It's not something I have done myself, so I am not 100% it will work that easily. Worth a shot!
